### PR TITLE
Sentence structure and Grammar in Evolution.md

### PR DESCRIPTION
**Pull request by CharlieS1103**

### CODE REVIEW

File: docs/project/evolution.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ This process is designed to:
 -   Ensure these kinds of changes can receive feedback from the entire
     community.
 -   Resolve questions and decide direction efficiently.
--   Create a clear log of rationale for why the project and language have
+-   Create a clear log of the rationale for why the project and language have
     evolved in particular directions.
 
-When there are questions, concerns, or issues with a proposal that need to be
+When there are questions, concerns, or issues with a proposal that needs to be
```
Comment (jonmeow) [inline docs/project/evolution.md]:
```suggestion
When there are questions, concerns, or issues with a proposal that need to be
```

I believe "need" is correct. https://oneminuteenglish.org/en/need-needs/

**Review by jonmeow — COMMENTED** [review]
Thanks! One small note, and I'm also going to let pre-commit run (we use prettier for formatting, it'll give formatting notes if needed: https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md#pre-commit)

**Comment by CharlieS1103** [comment]
@jonmeow Appreciate it! I'll look through the pre-commit docs, is there anything else you need me to do?

**Comment by chandlerc** [comment]
Nope. I kicked off another pre-commit run, you can see the output here: https://github.com/carbon-language/carbon-lang/runs/7479941477?check_suite_focus=true

Should be able to reproduce and fix with a local run based on the docs linked. =]

**Review by jonmeow — APPROVED** [review]
Thanks!

**Pull request merged**